src/main/java/org/snomed/snowstorm/core/data/services/BranchMetadataHelper.java

```diff
@@ -19,6 +19,8 @@
 	// Metadata for internal use only and can't be updated via branch metadata REST service
 	public static final String INTERNAL_METADATA_KEY = "internal";
 
+	public static final String AUTHOR_FLAGS_METADATA_KEY = "authorFlags";
```

Comment:
`BranchMetadataHelper` felt like the best location for the new `AUTHOR_FLAGS_METADATA_KEY` constant.